src/main/java/androidx/view/View.kt

```diff
@@ -171,3 +171,36 @@ fun View.toBitmap(config: Bitmap.Config = Bitmap.Config.ARGB_8888): Bitmap {
     }
     return Bitmap.createBitmap(width, height, config).applyCanvas(::draw)
 }
+
+/** Show current view */
+fun View.show() {
```

Comment:
`inline`

Reply:
@JakeWharton Is there a good rule when to inline?

Reply:
In this case you're aliasing to a public API and it doesn't require any allocation which is always a good candidate